packages/code/src/Code/Code.tsx

```diff
@@ -5,9 +5,12 @@ import PropTypes from 'prop-types';
 
 import { cx } from '@leafygreen-ui/emotion';
 import { useIsomorphicLayoutEffect } from '@leafygreen-ui/hooks';
+import ChevronDown from '@leafygreen-ui/icon/dist/ChevronDown';
+import ChevronUp from '@leafygreen-ui/icon/dist/ChevronUp';
```

Comment:
If we add transitions, then we could mimic Stephen's changes in #2443

Reply:
I like it, will do!

Reply:
I'm not sure the usage here is quite the same. This is primarily because both the text and the icon change on expansion/collapse. So rotating the icon alone doesn't inherently make the transition look less jarring. I have an idea and want to play around with a few possible solutions. I'll push those on separate commits before requesting another review, and probably run them by design to see what looks best here.